src/Components/v2/Lightbox/Lightbox.tsx

```diff
@@ -278,7 +279,7 @@ export class Lightbox extends React.Component<LightboxProps, LightboxState> {
   }
 
   render() {
-    const { children, enabled, isDefault, imageAlt } = this.props
+    const { children, enabled, isDefault, imageAlt, src } = this.props
```

Comment:
If I'm understanding correctly, this means that a `<Lightbox>` component will apply its `src` to all `<Image>` elements within it. I don't know much about the context of these changes, but that seems like a bug. 

From reading this change, I think this code:

```
<Lightbox src="image-1.jpg">
  <Image src="image-2.jpg" />
  <Image src="image-3.jpg" />
  <Image src="image-4.jpg" />
</Lightbox>
```

would result in `image-1.jpg` showing up three times, instead of seeing `image-2`, `image-3`, and `image-4`. 


Am I misunderstanding this change?

Reply:
Each `Lightbox` component renders only one image. Happy to look through it via a call if you want to delve into it. It's a complicated component.

Reply:
Summarizing a call between @kierangillen and I, for permanence: 

The `<Lightbox>` component is currently giving conflicting hints to devs. It iterates over all the children passed in, but (a) we are never actually giving it more than one, and (b) it doesn't seem like the second one is displayed very well (or at all, we aren't sure). With the changes in this PR, it now actually would iterate over all of them, but show the `src` applied to the `<Lightbox>` element each time.

We're working on finding out which is more accurate about the `<Lightbox>` - does it need to display more than one, in which case we need to remove these assumptions of only displaying one? Or does it only ever need to display one, in which case we can make it impossible to give this component more than one child image?

Reply:
Via Slack we've agreed to remove the `.map()` and render only one image per `Lightbox` component for now.